### PR TITLE
Removed dependency of transaction from doctrine

### PR DESCRIPTION
**Pull request by artempetrovcode**

### CODE REVIEW

File: Services/Transaction/Storage/Doctrine.php [hunk 1]
```diff
@@ -0,0 +1,153 @@
...
+     */
+    public function read($id) {
+        $values = [
+            'class'     => 'Ecentria\Libraries\EcentriaRestBundle\Entity\Transaction',
+            'converter' => 'ecentria.api.converter.entity',
+            'name'      => 'transactionEntity'
+        ];
+
+        $configuration = new ParamConverter($values);
+        $this->entityConverter->apply($this->request, $configuration);
```
Comment (artempetrovcode) [inline Services/Transaction/Storage/Doctrine.php]: @inferont, I had to pass Request object to the Storage in order to re-use EntityConverter::apply method.
That's not good. I'm thinking to use `$entityManager->find('Ecentria\Libraries\EcentriaRestBundle\Entity\Transaction', $id);` instead

File: Services/Transaction/Storage/Doctrine.php [hunk 2]
```diff
@@ -0,0 +1,118 @@
...
+    }
+
+    /**
+     * {@inheritDoc}
+     */
+    public function read($id) {
+        $entityManager = $this->registry->getManagerForClass(self::ENTITY_CLASS_NAME);
+        $transactionEntity = $entityManager->find(self::ENTITY_CLASS_NAME, $id);
+
+        return $this->buildModel($transactionEntity);
```
Comment (inferont) [inline Services/Transaction/Storage/Doctrine.php]: This isn't correctly handling a case when $transactionEntity was not found.

Comment (artempetrovcode) [inline Services/Transaction/Storage/Doctrine.php]: Handled

File: Services/Transaction/Storage/Doctrine.php [hunk 3]
```diff
@@ -0,0 +1,122 @@
...
+        return $transactionEntity;
+    }
+
+    /**
+     * Build Transaction Model from Transaction Entity
+     *
+     * @param TransactionEntity $transactionEntity
+     * @return TransactionModel
+     */
+    private function buildModel(TransactionEntity $transactionEntity)
```
Comment (maximTarleckiy) [inline Services/Transaction/Storage/Doctrine.php:104]: Why you split transaction to model and entity classes? Why can't use entity as model?

Comment (inferont) [inline Services/Transaction/Storage/Doctrine.php:104]: We're removing a dependency on doctrine for transactions. This model will be used with elastic search and monolog in the near future.

**Pull request merged**